llvm/lib/Target/Mips/MipsSEFrameLowering.cpp

```diff
@@ -935,21 +935,57 @@ bool MipsSEFrameLowering::assignCalleeSavedSpillSlots(
   if (!STI.hasNanoMips())
     return false;
 
+  std::unordered_map<unsigned, unsigned> Regs{
```

Comment:
While were at it, this map is constant, so can we add it as a static const member?